bundles/pom.xml

```diff
@@ -290,7 +291,7 @@
   </dependencies>
 
   <properties>
```

Comment:
remove if not needed

Reply:
This part is not owned/related to my binding, is it?

Reply:
correct